src/Couchbase.Lite.Shared/API/Query/Meta.cs

```diff
@@ -53,6 +53,16 @@ public static class Meta
         [NotNull]
         public static IMetaExpression Sequence => new QueryTypeExpression(SequenceKeyPath, SequenceColumnName);
 
+        /// <summary>
+        /// A metadata expression refering to the deleted boolean flag of the document.
+        /// </summary>
+        public static IMetaExpression IsDeleted => new QueryTypeExpression(SequenceKeyPath, SequenceColumnName);
+
```

Comment:
This is incorrect....it is just duplicating "sequence"

Reply:
Still not correct I think.  Double check with Pasin what this should be. I think it is `_deleted`

Reply:
Yes, the flag should be _deleted.